modules/compliance-unapplying-kubeletconfig.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-unapplying-kubeletconfig_{context}"]
+= Unapplying a KubeletConfig remediation
+`KubeletConfig` remediations are included in node-level profiles. In order to remove a KubeletConfig remediation, you must manually remove it from the  `KubeletConfig` object. This example demonstrates how to remove the compliance check for `one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available` remediation.
+
+.Procedure
+
+. Locate the `scan-name` and compliance check for the `one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available` remediation: 
++
+[source,terminal]
+----
+$ oc get remediation one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceRemediation
+metadata:
+  annotations:
+    compliance.openshift.io/xccdf-value-used: var-kubelet-evictionhard-imagefs-available
+  creationTimestamp: "2022-01-05T19:52:27Z"
+  generation: 1
+  labels:
+    compliance.openshift.io/scan-name: one-rule-tp-node-master <1>
+    compliance.openshift.io/suite: one-rule-ssb-node
+  name: one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available
+  namespace: openshift-compliance
+  ownerReferences:
+  - apiVersion: compliance.openshift.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: ComplianceCheckResult
+    name: one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available 
+    uid: fe8e1577-9060-4c59-95b2-3e2c51709adc
+  resourceVersion: "84820"
+  uid: 5339d21a-24d7-40cb-84d2-7a2ebb015355
+spec:
+  apply: true
+  current:
+    object:
+      apiVersion: machineconfiguration.openshift.io/v1
+      kind: KubeletConfig
+      spec:
+        kubeletConfig:
+          evictionHard:
+            imagefs.available: 10% <2>
+  outdated: {}
+  type: Configuration
+status:
+  applicationState: Applied
+----
+<1> The scan name of the remediation.
+<2> The remediation check for the remediation.
++  
+. Using the `scan-name`, find the `KubeletConfig` object that the remediation was applied to: 
++
+[source,terminal]
+----
+$ oc get kubeletconfig --selector compliance.openshift.io/scan-name=one-rule-tp-node-master
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                 AGE
+compliance-operator-kubelet-master   2m34s
+----
++  
+. Remove the remediation check from the KubeletConfig object by toggling the flag to false:
++
+[source,terminal]
+----
+$ $ oc patch complianceremediations/one-rule-tp-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available
+----
+
+. The remediation status will change to `NotApplied` and the composite `KubeletConfig` object will re-render to not include the remediation.
```

Comment:
Changing the remediation object to un-apply will not affect `KubeletConfig` object, once you find the name of Kubeletconfig, in this case `compliance-operator-kubelet-master`, you need to `oc edit KubeletConfig compliance-operator-kubelet-master`, and then remove <2> from the edit.

Also to note: if the user has scheduled scan that auto-apply remediation. they would need to exclude the rule they want to unapply in tailored profile, so they don't get applied again.